pontoon/localizations/views.py

```diff
@@ -89,7 +89,7 @@ def ajax_resources(request, code, slug):
         pages[page.name] = latest_page_translatedresource
 
     translatedresources = {s.resource.path: s for s in translatedresources_qs}
-    translatedresources = dict(translatedresources.items() + pages.items())
+    translatedresources = dict(list(translatedresources.items()) + pages.items())
```

Comment:
I've seen a few of those `list()` added. Why is that needed?

Reply:
You can't concat `dict_items`. Interestingly enough, this is not enough, both `items` need to be converted.

Or you create a dict from `translatedresources` first, and then update from `pages`.

Reply:
Ah, `.keys()` doesn't return a simple `list`? I didn't know that. Thanks!

Reply:
@Pike Thanks :) Added to my todo list.